js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -460,7 +519,7 @@ export class OpenAIModel implements PromptCompletionModel {
         };
     }
 
-    private returnError(err: unknown, input: Message<string>|undefined): PromptResponse<string> {
+    private returnError(err: unknown, input: Message<string> | undefined): PromptResponse<string> {
```

Comment:
this method could be refactored to be much more readable by flattening like so:

```typescript
    private returnError(err: unknown, input: Message<string> | undefined): PromptResponse<string> {
        if (err instanceof OpenAI.APIError) {
            if (err.status == 429) {
                if (this.options.logRequests) {
                    console.log(Colorize.title('HEADERS:'));
                    console.log(Colorize.output(err.headers as any));
                }
                
                return {
                    status: 'rate_limited',
                    input,
                    error: new Error(`The chat completion API returned a rate limit error.`)
                };
            }
            
            return {
                status: 'error',
                input,
                error: new Error(`The chat completion API returned an error status of ${err.status}: ${err.name}`)
            };
        }
        
       return {
          status: 'error',
          input,
          error: new Error(`The chat completion API returned an error: ${(err as Error).toString()}`)
       };
    }
```

ie the `else` statements are not necessary because you are `returning`